flexmeasures/data/schemas/forecasting/pipeline.py

```diff
@@ -20,45 +20,157 @@ class TrainPredictPipelineConfigSchema(Schema):
 
 class ForecasterParametersSchema(Schema):
 
-    sensor = SensorIdField(required=True)
+    sensor = SensorIdField(
+        required=True,
+        metadata={
+            "description": "ID of the sensor to forecast.",
+            "example": 2092,
+        },
+    )
     future_regressors = fields.List(
         SensorIdField(),
         required=False,
+        metadata={
+            "description": "Sensor IDs to be treated only as future regressors.",
+            "example": [2093, 2094],
+        },
     )
     past_regressors = fields.List(
         SensorIdField(),
         required=False,
+        metadata={
+            "description": "Sensor IDs to be treated only as past regressors.",
+            "example": [2095],
+        },
     )
     regressors = fields.List(
         SensorIdField(),
         required=False,
+        metadata={
+            "description": "Sensor IDs used as both past and future regressors.",
+            "example": [2093, 2094, 2095],
+        },
+    )
+    model_save_dir = fields.Str(
+        required=True,
+        metadata={
+            "description": "Directory to save the trained model.",
+            "example": "flexmeasures/data/models/forecasting/artifacts/models",
+        },
+    )
+    output_path = fields.Str(
+        required=False,
+        allow_none=True,
+        metadata={
+            "description": "Directory to save prediction outputs. Defaults to None (no outputs saved).",
+            "example": "flexmeasures/data/models/forecasting/artifacts/forecasts",
+        },
+    )
+    start_date = AwareDateTimeOrDateField(
+        required=False,
+        allow_none=True,
+        metadata={
+            "description": "Timestamp marking the start of training data. Defaults to train_period before start_predict_date if not set.",
+            "example": "2025-01-01T00:00:00+01:00",
+        },
+    )
+    end_date = AwareDateTimeOrDateField(
+        required=True,
+        inclusive=True,
+        metadata={
+            "description": "End date for running the pipeline.",
+            "example": "2025-10-15T00:00:00+01:00",
+        },
+    )
+    train_period = DurationField(
+        required=False,
+        allow_none=True,
+        metadata={
+            "description": "Duration of the initial training period (ISO 8601 format, min 2 days). If not set, derived from start_date and start_predict_date or defaults to P30D (30 days).",
+            "example": "P7D",
+        },
+    )
+    start_predict_date = AwareDateTimeOrDateField(
+        required=False,
+        allow_none=True,
+        metadata={
+            "description": "Start date for predictions. Defaults to now floored to sensor resolution.",
```

Comment:
```suggestion
            "description": "Start date for predictions. Defaults to now, floored to the sensor resolution, so that the first forecast is about the ongoing event.",
```